File: Week1/roman-numerals/js/romanNumeralsSpec.js
var rn = require("./romanNumerals");

const romansLazy = rn.toRomansLazy
const romansModern = rn.toRomansModern

console.log(romansLazy(1) === 'I');
console.log(romansLazy(3) === 'III');
console.log(romansLazy(4) === 'IIII');


console.log(romansModern(4) === 'IV')
console.log(romansModern(944) === 'CMXLIV')
console.log(romansModern(150) === 'CL')

File: Week1/roman-numerals/ruby/roman_numerals_spec.rb
require "./roman_numerals.rb"

puts(toRomansLazy(1) == 'I')
puts(toRomansLazy(3) == 'III')
puts(toRomansLazy(4) == 'IIII')

puts(toRomansModern(4) == 'IV')
puts(toRomansModern(944) == 'CMXLIV')
puts(toRomansModern(150) == 'CL')